### ISSUE TITLE
Cannot start remote ac

### ISSUE THREAD

**Issue by agarest-lab** [SAIC-iSmart-API/saic-python-mqtt-gateway#63]
![photo1690864670](https://github.com/SAIC-iSmart-API/saic-python-mqtt-gateway/assets/76034979/7431204d-733d-4a06-83ee-4911e1fd5135)

**Comment by manu78** [comment]
I have another error in my MG4 integration

```
2023-10-27 06:20:37,585 A/C will be switched on
2023-10-27 06:20:37,585 Error while processing MQTT message: temperature_idx must be between 0 and 14
Traceback (most recent call last):
  File "/usr/src/app/mqtt_publisher.py", line 70, in __on_message
    self.__on_message_real(client, userdata, msg)
  File "/usr/src/app/mqtt_publisher.py", line 91, in __on_message_real
    self.on_mqtt_command_received(vin, msg)
  File "/usr/src/app/./mqtt_gateway.py", line 316, in __on_mqtt_command_received
    vehicle_handler.handle_mqtt_command(msg)
  File "/usr/src/app/./mqtt_gateway.py", line 181, in handle_mqtt_command
    self.saic_api.start_ac(self.vin_info, temperature_idx=self.vehicle_state.get_ac_temperature_idx())
  File "/usr/local/lib/python3.10/site-packages/saic_ismart_client/saic_api.py", line 277, in start_ac
    return self.control_climate(vin_info, fan_speed=2, ac_on=None, temperature_idx=temperature_idx)
  File "/usr/local/lib/python3.10/site-packages/saic_ismart_client/saic_api.py", line 324, in control_climate
    raise Exception('temperature_idx must be between 0 and 14')
Exception: temperature_idx must be between 0 and 14
```

**Comment by nanomad** [comment]
@tosate I think it's best to remove the saic_api.py check, on an MG4 the idx can go up to 19 (33C or MAX as shown in the app)

**Comment by nanomad** [comment]
Fixed in 0.5.x, feel free to re-open if it still stands
